src/app/components/Transactions/LogEvent.tsx

```diff
@@ -0,0 +1,137 @@
+import { EvmEventParam, Layer, RuntimeEvent, RuntimeEventType } from '../../../oasis-indexer/api'
+import React, { FC } from 'react'
+import { Link as RouterLink } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import { StyledDescriptionList } from '../StyledDescriptionList'
+import { useTheme } from '@mui/material/styles'
+import Divider from '@mui/material/Divider'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import Typography from '@mui/material/Typography'
+import { RouteUtils } from '../../utils/route-utils'
+import Link from '@mui/material/Link'
+import Table from '@mui/material/Table'
+import TableHead from '@mui/material/TableHead'
+import TableRow from '@mui/material/TableRow'
+import TableCell from '@mui/material/TableCell'
+import TableBody from '@mui/material/TableBody'
+import { AccountLink } from '../Account/AccountLink'
+import { CopyToClipboard } from '../CopyToClipboard'
+
+const EvmEventParamData: FC<{ layer: Layer; param: EvmEventParam }> = ({ layer, param }) => {
+  switch (param.evm_type) {
+    case 'address':
+      return <AccountLink address={param.value as string} layer={layer} />
```

Comment:
We should probably move monospace styling into `AccountLink`

Reply:
Done.

Reply:
Re: monostace vs AccountLink: see also https://github.com/oasisprotocol/explorer/pull/225